src/components/BlockFeatured/BlockFeatured.vue

```diff
@@ -122,6 +160,42 @@ export default {
         },
         formattedCredits() {
             return nl2br(this.credits).split("<br>")
+        },
+        formattedCreditsSecondary() {
+            return nl2br(this.creditsSecondary).split("<br>")
+        }
```

Comment:
You don't need `nl2br` here if you are doing the split. You can just do `this.credits.split(/(?:\r\n|\r|\n)/g)`